Dockerfile

```
@@ -8,11 +8,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+FROM public.ecr.aws/eks-distro/kubernetes/go-runner:v0.13.0-eks-1-23-1 as go-runner
```

Can you parameterize this?
```suggestion
ARG GO_RUNNER=public.ecr.aws/eks-distro/kubernetes/go-runner:v0.13.0-eks-1-23-1
FROM ${GO_RUNNER} as go-runner
```